### ISSUE TITLE
Add auto-update of WebSocket view

### ISSUE THREAD

**Issue by fkromer** [a85/POSTMan-Chrome-Extension#10308]
### Is there an existing request for this feature?

- [X] I have searched the existing issues for this feature request

### Is your feature request related to a problem?

Received messages are listed in the "Messages" area of the WebSocket Request building block. In many situations it's important to beeing able to investigate message payload content over time. This is not possible ATM.

### Describe the solution you'd like

It would be valuable to have the option to show the payload of the most recent message received in an "auto-update" view.

### Describe alternatives you've considered

Using another tool istead.

**Comment by codenirvana** [comment]
@fkromer from "auto-update" do you mean displaying the entire message payload instead of the current collapsed state in the message stream?

**Comment by fkromer** [comment]
@codenirvana Yes, and that the location of the view showing the message payload is kept static.
